parsers/OPENNEM.py

```diff
@@ -163,9 +196,10 @@ def fetch_production(zone_key=None, session=None, target_datetime=None, logger=l
         },
         'source': SOURCE,
         'zoneKey': zone_key,
-    } for dt, row in df.iterrows()]
+    } for dt, row in tqdm(df.iterrows())]
```

Comment:
Used tqdm to print the speed of this iteration (it's currently the bottleneck in the parser)